lib/mayaUsd/fileio/jobs/readJob.cpp

```diff
@@ -602,85 +844,83 @@ void UsdMaya_ReadJob::_ImportPrototype(
     const UsdPrim&            usdRootPrim,
     UsdMayaPrimReaderContext& readCtx)
 {
-    _PrimReaderMap     primReaderMap;
     const UsdPrimRange range = UsdPrimRange::PreAndPostVisit(prototype);
-    for (auto primIt = range.begin(); primIt != range.end(); ++primIt) {
-        const UsdPrim&           prim = *primIt;
-        UsdMayaPrimReaderContext readCtx(&mNewNodeRegistry);
-        readCtx.SetTimeSampleMultiplier(mTimeSampleMultiplier);
-        if (prim.IsInstance()) {
-            _DoImportInstanceIt(primIt, usdRootPrim, readCtx, primReaderMap);
-        } else {
-            _DoImportPrimIt(primIt, usdRootPrim, readCtx, primReaderMap);
-        }
-    }
+    _ImportPrimRange(range, usdRootPrim);
```

Comment:
While in import, I refactor some code that was duplicated and moved the instance prototype cleanup to its own function for clarity.